etc/kube/pachyderm.json

```diff
@@ -381,6 +381,16 @@
                 "mountPath": "/pach"
               }
             ],
+            "readinessProbe": {
+              "exec": {
+                "command": [
+                  "./pachd",
+                  "--readiness-check"
+                ]
+              },
+              "initialDelaySeconds": 15,
+              "timeoutSeconds": 1
+            },
```

Comment:
This file is actually autogenerated by `pach-deploy` check in `src/server/pkg/deploy/assets/assets.go` for where you should add it.